### ISSUE TITLE
Shea Craig's JSS Repo Listing

### ISSUE THREAD

**Issue by jmb03012** [lindegroup/autopkgr#204]
When entering values for the JSS section for the first time AutoPkgr automatically adds and selects Shea Craig's JSS Recipes Repo for you.

Over the course of my testing, there have been some occasions where I've had to purge those settings in the JSS section, remove his repo, edit (or fully delete the plist), etc, all which results in the added repo disappearing.

If I go an re-fill out the JSS section it does of course continue to work, but only after I manually add his repo back for the JSS recipes.  Would be a nice to have if it either was in the default list of recipes, or anytime you fill out the JSS section details it does a check if that repo is missing and re-adds it (even if you had to toggle it) since its such a core part of that initial functionality.
